orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -56,7 +56,9 @@ def _forward(self, *inputs, **kwargs):
         # inside _PytorchModuleMetadata
         self._module_metadata = _PytorchModuleMetadata(module, _io._FlattenedModule(module))
 
-        self._execution_manager = GraphExecutionManagerFactory(self._module_metadata.flattened_module)
+        onnx_export_type = torch.onnx.OperatorExportTypes.ONNX
```

Comment:
As I mentioned in another comment, may we not populate ```ORTModule``` with configuration settings? We can put this inside ```graph_execution_manager``` with a default value and leverage the experimental functions to change the ```onnx_export_type``` (similar to the loglevel) until we have an advanced way to change the configurations for ```ORTModule```.

Reply:
Yep. Already done in lastest PR. Thanks.